tests/ruby/std_streams_spec.rb

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env rspec
+
+require_relative "test_helper"
+require "yast/ui_shortcuts"
+
+Yast.import "UI"
+
+# We do not have a proper ncurses in travis at the moment
+if !ENV["TRAVIS"]
```

Comment:
I do not like much complete enclosing whole file. What about `exit 0 if ENV["TRAVIS"]` ?